simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -101,6 +99,72 @@ public StandardExports() {
     return mfs;
   }
 
+  static Long callLongGetter(String getterName, Object obj) throws NoSuchMethodException {
+    Method method = obj.getClass().getMethod(getterName);
+    return (Long) callMethod(method, obj).getValue();
+  }
+
+  /**
+   * Attempts to call a method either directly or via one of the implemented interfaces.
+   */
+  static ReturnValue callMethod(Method method, Object obj) {
+    try {
+      method.setAccessible(true);
```

Comment:
Great point. Removed, as we don't need it for our current uses.

One observation though: if we ever want to invoke a method that's private/protected (because it's only declared in the concrete implementation, not an interface) it will be necessary to add it back.